.gitmodules

```diff
@@ -1,3 +1,3 @@
 [submodule "cmake"]
 	path = cmake
-	url = https://github.com/jrl-umi3218/jrl-cmakemodules.git
+	url = https://github.com/jorisv/jrl-cmakemodules.git
```

Comment:
Any reason to do this?

Reply:
It's linked to this pull request : https://github.com/jrl-umi3218/jrl-cmakemodules/pull/701
submodule will be changed when jrl-cmakemodules PR will be merged.